src/main/java/org/opentripplanner/model/TransitMode.java

```diff
@@ -10,5 +12,16 @@
   AIRPLANE,
   CABLE_CAR,
   GONDOLA,
-  FUNICULAR
+  FUNICULAR,
+  TROLLEYBUS,
+  MONORAIL;
+
+  static final EnumSet<TransitMode> BUS_TYPE_MODES = EnumSet.of(
+          TransitMode.BUS, TransitMode.TROLLEYBUS
+  );
```

Comment:
```suggestion
  private static final EnumSet<TransitMode> ON_STREET_MODES = EnumSet.of(
         COACH, BUS, TROLLEYBUS
  );
```
- Use `private` not package local
- Naming, what is the business logic focusing on? In this case it is the fact that it is running in the street network.
- DRY, The `TransitMode.BUS` repeat the context of the class.